assets/src/scripts/components/hydrograph/time-series-graph.js

```diff
@@ -196,6 +255,12 @@ export const drawTimeSeriesGraph = function(elem, store, siteNo, showMLName, sho
             xscale: getMainXScale('current'),
             yscale: getMainYScale,
             seriesPoints: getCurrentVariableMedianStatPoints
+        })))
+       .call(link(store, plotAllFloodLevelPoints, createStructuredSelector({
+            visible: waterwatchVisible,
+            xscale: getBrushXScale('current'),
```

Comment:
Shouldn't this be getMainXScale for this graph?

Reply:
Using getBrushXScale updates the dates of the two points with the minimal amount of code in the case of a brush event.  getMainXScale doesn't address brush conditions.  All the tests pass and smoke tests seem OK but let me know if there is anything I'm not considering.

Reply:
But we are graphing the lines on the main graph. It only appears to work because enableClip is true. I think. At any rate, it seems like if we are using the getMainXScale to scale the time series itself that we should be doing the same thing here. The mainXScale is the one that changes when the brush selection is changed.

Reply:
Sounds good, we don't want a fail on enabledClip false.  So here is a picture of the graph when using getXMainScale:
![Capture](https://user-images.githubusercontent.com/30809186/83303671-79ac7600-a1cb-11ea-8a61-716f86b316f6.PNG)
The issue exists for the median stats as well.  They run off the side of either graph when brushed because the offsets refer to time series sampled at higher resolutions.  Is addressing within the scope of the ticket?  I don't want to assume so.

Reply:
![Capture2](https://user-images.githubusercontent.com/30809186/83304442-deb49b80-a1cc-11ea-9baa-e2c6c28ea2d2.PNG)
Sorry I cutoff the brush portion by mistake.  This only happens after a brush event that filters some of the time series.